File: Web.Admin/src/api/services/account/models/ChangePasswordInput.ts
/**
 * Fast.Center.Service.Account.Dto.ChangePasswordInput 账号修改密码输入
 */
export interface ChangePasswordInput {
  /**
   * 旧密码
   */
  oldPassword?: string;
  /**
   * 新密码
   */
  newPassword?: string;
  /**
   * 确认密码
   */
  confirmPassword?: string;
  /**
   * 更新版本控制字段
   */
  rowVersion?: number;
}

File: Api.Server/src/Services/Center/Account/Dto/ChangePasswordInput.cs
namespace Fast.Center.Service.Account.Dto;

/// <summary>
/// <see cref="ChangePasswordInput"/> 账号修改密码输入
/// </summary>
public class ChangePasswordInput : UpdateVersionInput
{
    /// <summary>
    /// 旧密码
    /// </summary>
    [StringRequired(ErrorMessage = "旧密码不能为空")]
    public string OldPassword { get; set; }

    /// <summary>
    /// 新密码
    /// </summary>
    [StringRequired(ErrorMessage = "新密码不能为空"), MinLength(6, ErrorMessage = "新密码不能少于6位字符")]
    public string NewPassword { get; set; }

    /// <summary>
    /// 确认密码
    /// </summary>
    [StringRequired(ErrorMessage = "确认密码不能为空"), MinLength(6, ErrorMessage = "确认密码不能少于6位字符")]
    public string ConfirmPassword { get; set; }
}